src/js/components/PageHeader/stories/MultipleActions.js

```diff
@@ -67,6 +67,7 @@ export const MultipleActions = () => {
             </Box>
           }
           parent={<Anchor label="Parent Page" />}
+          level="1"
```

Comment:
We shouldn't have our examples pass a prop that's already basically the default (and we don't want these examples to be some other level). This comment goes for all the other story changes too.

Instead there should probably be a new story that shows use of `level`

Reply:
@MikeKingdom what should be the structure of the new story that shows the use of level. Can you please elaborate it. Should it be like the ```size``` story where first four levels can be used to give an example of how the level prop works.

Reply:
> @MikeKingdom what should be the structure of the new story that shows the use of level. Can you please elaborate it. Should it be like the `size` story where first four levels can be used to give an example of how the level prop works.

I think something similar to the Size story. Try to keep it like something realistic rather than just a `level` testcase